tests/components/sharkiq/test_shark_iq.py

```diff
@@ -175,9 +169,8 @@ async def test_shark_metadata(hass: HomeAssistant) -> None:
         "model": "RV1001AE",
         "sw_version": "Dummy Firmware 1.0",
     }
-    state_json = json.dumps(shark.device_info, sort_keys=True)
-    target_json = json.dumps(target_device_info, sort_keys=True)
-    assert state_json == target_json
+
+    assert shark.device_info == target_device_info
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6103615)